lightning/src/routing/router.rs

```diff
@@ -412,7 +412,7 @@ impl<'a> CandidateRouteHop<'a> {
 	fn effective_capacity(&self) -> EffectiveCapacity {
 		match self {
 			CandidateRouteHop::FirstHop { details } => EffectiveCapacity::ExactLiquidity {
-				liquidity_msat: details.outbound_capacity_msat,
+				liquidity_msat: details.next_outbound_htlc_limit_msat,
```

Comment:
Might we also want to prefer `next_outbound_htlc_limit_msat` when sorting channels (L844)?

Also there's a mention of `outbound_capacity_msat` on L821 where docs may need updating